charts/catalog/README.md

```diff
@@ -42,7 +42,9 @@ chart and their default values.
 |-----------|-------------|---------|
 | `image` | apiserver image to use | `quay.io/kubernetes-service-catalog/service-catalog:v0.1.33` |
 | `imagePullPolicy` | `imagePullPolicy` for the service catalog | `Always` |
-| `deploymentStrategy` | `deploymentStrategy` for the service catalog deployments | `RollingUpdate` |
+| `apiserver.replicas` | `replicas` for the service catalog apiserver pod count | `2` |
```

Comment:
I think the default should be 1

Reply:
I'm happy to change the recommended default to 1 for both apiserver.replicas and controllerManager.replicas, but in my testing, I found 1 for both wasn't enough as the apiserver and controllerManager would repeatedly crash/restart. The stability seemed to improve with multiple replicas. I opened an issue [here](https://github.com/kubernetes-incubator/service-catalog/issues/2380) for that as well. Do you have insight into that?

Also, by adding the option for replicas, even if users aren't using replicasets at this time, the option in the framework/template sets up scalability as an option. 

I will update the PR to include 1 as minimum and notes about replicas in production scenarios.

Reply:
@jldeen Yeah, I too have seen those but last I asked around it seemed like I was the only one.  I updated the other issue with a comment, perhaps timeoutSeconds needs to be relaxed.  I'll give that a shot next time I'm in there.

I absolutely agree about support with replicas, I think its a good option to expose.

Reply:
You're definitely not the only one, but funny enough, I worried I was too since there wasn't an issue about it. I tested your theory and disabled (temporarily) health check, updated the deployment and there have now been 0 restarts in 2 hours for both apiServer and controllerManager. I will make the requested changes to PR for the proposed updates and then submit a new PR to solve my open issue after I tweak the timeoutSeconds and checks accordingly.

Thanks!